### DISCUSSION TITLE
Reference to type "DependencyObjectCollection" claims it is define in "uno.ui", but it could not be found

### DISCUSSION THREAD

**Discussion by Guidemarcus** [unoplatform/uno#10879] [Q&A] [answered]
I am updating some behaviors to net6 and need to use Microsoft.Xaml.Interactivity to access the Behavior class. I saw that this, https://www.nuget.org/packages/Uno.Microsoft.Xaml.Behaviors.Interactivity/2.3.0?_src=template#dependencies-body-tab, exists, but when using it I see this error

Reference to type "DependencyObjectCollection" claims it is define in "uno.ui", but it could not be found

 
Would it be because it is referencing uno.ui instead of uno.winui? Does this error sounds any bell?

**Comment by francoistanguay** [comment, marked as answer]
If you're targetting Uno.WinUI, then you need to use the WinUI version: https://www.nuget.org/packages/Uno.Microsoft.Xaml.Behaviors.Interactivity.WinUI/2.3.1-uno.2
